include/fiber_backend_swfft.h

```diff
@@ -9,7 +9,9 @@
 #include <stdio.h>
 
 #if defined(FIBER_ENABLE_SWFFT)
-#include "swfft.h"
+#include "fiber_swfft.h"
```

Comment:
We should keep original library name: SWFFT

Reply:
There is no "original library".  See above.